lib/tasks/data_migrations.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+namespace :data_migrations do
```

Comment:
Our data migrations run automatically in production and staging so no need for a rake task.